akka-http-tests/src/test/scala/akka/http/scaladsl/server/DontLeakActorsOnFailingConnectionSpecs.scala

```diff
@@ -59,22 +60,25 @@ abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String
             handleResponse(resp, id)
         }
 
-        val running = source.via(clientFlow).runWith(sink)
-        countDown.await(10, TimeUnit.SECONDS) should be(true)
-        Await.result(running, 10.seconds)
-
+        EventFilter.error(pattern = """\d+\: FAIL.*""", occurrences = reqsCount).intercept {
```

Comment:
We can keep that change but it's not strictly necessary as all log messages will be filtered anyway if the test succeeds and otherwise, all will be shown if the test fails.

Reply:
The EventFilters would still validate the error message count.

Reply:
Though, that might not be reliable anymore, because WithLogCapturing will install a filter that will filter out all messages and depending on the order of filters, one or the other might not even be called.

Reply:
I thought I resolved the conflict  by removing these changes, must have done something wrong in IntelliJ somehow